sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -139,7 +139,7 @@
         }
         versionCreated = getVersionCreated(mappingMap);
         versionUpgraded = getVersionUpgraded(mappingMap);
-        closed = getClosed(mappingMap);
+        closed = metaData.getState().equals(IndexMetaData.State.CLOSE);
```

Comment:
as this are enums pls change to `metaData.getState() == IndexMetaData.State.CLOSE`